specutils/utils/wcs_utils.py

```diff
@@ -85,6 +90,11 @@ def refraction_index(wavelength, method='Griesen2006', co2=None):
     VALID_METHODS = ['Griesen2006', 'Edlen1953', 'Edlen1966', 'Morton2000',
                      'PeckReeder1972', 'Ciddor1996']
     assert isinstance(method, str), 'method must be a string'
+    if method != 'Griesen2006' and wavelength.min() < 250 * u.nm:
```

Comment:
```suggestion
    if method != 'Griesen2006' and wavelength.min() < 200 * u.nm:
```
The literature puts the limit for this method at 200 nm, might as well match that.